src/sql/magic_cmd.py

```diff
@@ -210,6 +211,81 @@ def execute(self, cmd_name="", others="", cell="", local_ns=None):
 
             return report
 
+        elif cmd_name == "snippets":
```

Comment:
this if statement is becoming longer and longer. please move the logic into a different file: sql/sqlcmd.py so the if statement becomes:

```python
elif cmd_name == "snippets":
    sqlcmd_snippets()
```

Reply:
Should I move the logic for the other command as well? e.g., tables/columns

Reply:
just move the logic from this query, I opened an issue to move the other ones